values.yaml

```
@@ -458,7 +460,7 @@ rabbitmq-ha:
   #rabbitmqMemoryHighWatermark: 512MB
   #rabbitmqMemoryHighWatermarkType: absolute
   # Up to 255 character string, should be fixed so that re-deploying the chart does not fail (see: https://github.com/helm/charts/issues/12371)
```

As we're defaulting `rabbitmqErlangCookie` value for everyone, let's at least include a warning comment as recommendation to change the default.

```suggestion
  # Up to 255 character string, should be fixed so that re-deploying the chart does not fail (see: https://github.com/helm/charts/issues/12371)
  # NB! It's highly recommended to change the default insecure rabbitmqErlangCookie value!
```

---

Warning is added as well